packages/react-ai/src/components/AIConversation/views/default/Attachments.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { Button, Image, Text, View } from '@aws-amplify/ui-react';
```

Comment:
Why aren't we using the elements API for these?

Reply:
These are the default amplify components which use our primitives